frontend/src/pages/patients/PatientsPage.tsx

```diff
@@ -104,17 +104,19 @@ const NO_PHI_SEARCH_RESULTS = {
     "No results were found for your search. No patient IDs in your search exist in either the SMILE or CRDB databases.",
 };
 
+interface IPatientsPageProps {
+  userEmail: string | null;
+  setUserEmail: Dispatch<SetStateAction<string | null>>;
+}
+
 export default function PatientsPage({
   userEmail,
   setUserEmail,
-}: {
-  userEmail: string | null;
-  setUserEmail: Dispatch<SetStateAction<string | null>>;
-}) {
```

Comment:
Moved prop definitions into a named interface to match how we do it in other places (e.g. `RecordsList.tsx`).